.gitignore

```diff
@@ -0,0 +1,13 @@
+.vagrant/machines/default/virtualbox/action_provision
```

Comment:
Hard coding the file names is not a good practice nor it is useful. What if a new vagrant version introduce a new file name in the cache directory?